src/common/utils/url.ts

```diff
@@ -79,10 +79,14 @@ function getServerInfo(serverUrl: URL | string) {
 
     // does the server have a subpath?
     const pn = parsedServer.pathname.toLowerCase();
-    const subpath = pn.endsWith('/') ? pn.toLowerCase() : `${pn}/`;
+    const subpath = getFormattedPathName(pn);
     return {subpath, url: parsedServer};
 }
 
+export function getFormattedPathName(pn: string) {
+    return pn.endsWith('/') ? pn.toLowerCase() : `${pn}/`;
```

Comment:
why do we only make it lowercase if it ends with '/'?

Reply:
I copied this from above, not sure why it was done that way.
https://github.com/mattermost/desktop/pull/1680/files/940a6f4803317dda9ea7d18f671bba1040b2e25f#diff-abddc5347f7e36fa964b14961781c90b425e015942e189a1bd61f09d6105aad6L82

Reply:
we should probably check for ending in `/` and make it lowercase in both places, most likely there is no problem with that, but if we ever run into a problem caused by that we are going to be soo confused 😆